src/pages/publication/MyPublications.tsx

```diff
@@ -42,9 +50,18 @@ const MyPublications: FC = () => {
     loadData();
   }, [dispatch]);
 
+  const userName = user.name;
+
   return (
     <FormCard>
       <FormCardHeading>{t('workLists:my_publications')}</FormCardHeading>
+      <StyledButtonWrapper>
+        <MuiLink component={Link} to={`/public-profile/${userName}`}>
```

Comment:
det blir noen stygge url'er med fullt navn med mellomrom (%20). Kan vi bruke en id eller noe annet ?

Reply:
jeg synes det er bedre å vise en stygg url isteden for å expose id'en til folk

Reply:
personnavn er uansett ikke entydige så vi må gjøre noe med denne

Reply:
I wireframen er det også koblet til institusjon `/public-profile/<institution>/<userName>`, men ser ikke hvordan vi kan gjøre det på en enkel måte.

Reply:
keep it simple - for now. neppe noen spec på dette